pkg/k8shandler/status.go

```diff
@@ -156,18 +165,40 @@ func (clusterRequest *ClusterLoggingRequest) getElasticsearchStatus() ([]logging
 	}
 
 	if len(esList.Items) != 0 {
-		for _, node := range esList.Items {
+		for _, cluster := range esList.Items {
+
+			nodeConditions := make(map[string][]elasticsearch.ClusterCondition)
 
 			nodeStatus := logging.ElasticsearchStatus{
-				ClusterName:   node.Name,
-				NodeCount:     node.Spec.Nodes[0].NodeCount,
-				Deployments:   getDeploymentNames(node.Status),
-				ReplicaSets:   getReplicaSetNames(node.Status),
-				StatefulSets:  getStatefulSetNames(node.Status),
-				Pods:          getPodMap(node.Status),
-				ClusterHealth: node.Status.ClusterHealth,
+				ClusterName:            cluster.Name,
+				NodeCount:              cluster.Spec.Nodes[0].NodeCount,
+				ClusterHealth:          cluster.Status.ClusterHealth,
+				Cluster:                cluster.Status.Cluster,
+				ShardAllocationEnabled: cluster.Status.ShardAllocationEnabled,
+				Pods:              getPodMap(cluster.Status),
```

Comment:
formatting

Reply:
weird... that is with `gofmt`

Reply:
moved things around and it formatted it better...